src/app/templates/_bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "<%= nameOf.package %>",
+  "version": "0.0.0",
```

Comment:
Yeah, that's why I used 0.0.0 - because then you can't (easily) use `npm version major` for your first release.